ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,458 +1,470 @@
+{%                 set neighbor_default_originate_cli = "neighbor " ~ neighbor ~ " default-originate" %}
+{%                 if router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.route_map is arista.avd.defined %}
+{%                     set neighbor_default_originate_cli = neighbor_default_originate_cli ~ " route-map " ~ router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.route_map %}
+{%                 endif %}
+{%                 if router_bgp.address_family_ipv4.neighbors[neighbor].default_originate.always is arista.avd.defined(true) %}
+{%                     set neighbor_default_originate_cli = neighbor_default_originate_cli ~ " always" %}
+{%                 endif %}
+      {{ neighbor_default_originate_cli }}
 {%             endif %}
 {%             if router_bgp.address_family_ipv4.neighbors[neighbor].activate == true %}
       neighbor {{ neighbor }} activate
 {%             elif router_bgp.address_family_ipv4.neighbors[neighbor].activate == false %}
       no neighbor {{ neighbor }} activate
 {%             endif %}
-{%          endfor %}
-{%       endif %}
+{%         endfor %}
 {%     endif %}
 {# address family ipv4 multicast activation #}
-{%     if router_bgp.address_family_ipv4_multicast is defined and router_bgp.address_family_ipv4_multicast is not none %}
+{%     if router_bgp.address_family_ipv4_multicast is arista.avd.defined %}
    !
    address-family ipv4 multicast
-{%       if router_bgp.address_family_ipv4_multicast.peer_groups is defined and router_bgp.address_family_ipv4_multicast.peer_groups is not none %}
-{%          for peer_group in router_bgp.address_family_ipv4_multicast.peer_groups | arista.avd.natural_sort %}
-{%             if router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].route_map_in is defined and router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].route_map_in is not none %}
+{%         for peer_group in router_bgp.address_family_ipv4_multicast.peer_groups | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].route_map_in is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].route_map_in }} in
 {%             endif %}
-{%             if router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].route_map_out is defined and router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].route_map_out is not none %}
+{%             if router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].route_map_out is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].route_map_out }} out
 {%             endif %}
 {%             if router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].activate == true %}
```

Comment:
No check if `router_bgp.address_family_ipv4_multicast.peer_groups[peer_group].activate` is defined